docs/releases/4.0.md

```diff
@@ -56,6 +56,34 @@ In Wagtail 2.16, we introduced support for Windows High Contrast mode (WHCM). Th
 * Display the correct color for icons in forced colors mode
 * Add a border around modal dialogs so they can be identified in forced colors mode
 
+### UX unification and consistency improvements
+
+In Wagtail 3.0, a new Page Editor experience was introduced, this release brings many of the UX and UI improvements to other parts of Wagtail for a more consistent experience.
+The bulk of these enhancements have been from Paarth Agarwal, who has been doing the [UX Unification](https://github.com/wagtail/wagtail/discussions/8158) Google Summer of Code project, sponsored by Torchbox with support from LB (Ben Johnston), Thibaud Colas and Helen Chapman.
```

Comment:
Just wanted to check - is it correct to refer to this as a Google Summer of Code project given that the sponsorship in this case was from Torchbox rather than through Google?

Other than that, all looks good to me!

Reply:
good call out.